src/main/java/net/mcreator/element/types/Block.java

```diff
@@ -182,7 +182,7 @@ public Block(ModElement element) {
 	}
 
 	public int renderType() {
-		if (blockBase != null && !blockBase.equals(""))
+		if (blockBase != null && !blockBase.equals("") && !blockBase.equals("Leaves"))

```

Comment:
One more question. Why was this added? 

This is here only to handle broken definitions so I don't think you should fallback to -1 in case there is anything else but Leaves.

Reply:
Ah, my bad. It was to make leaves have the same preview as single texture, but I guess I can move the check to line 208